_data/accessibility-tests/file-input.yml

```diff
@@ -0,0 +1,103 @@
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.1
+    wcag_criterion: 2.5.7
+  - summary: Content doesn't change until the user takes an action to change it.
+    summary_additional: When you access the file input with a keyboard, content stays the same until you use keys to make changes.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.1
+    wcag_criterion: 3.2.2
+# Screen reader tests
+  - summary: Screen reader announces file input.
+    summary_additional: When you navigate with arrow keys and the tab key while using a screen reader, you hear the prompt to "drag file here or choose from folder" and you hear any related hints or instructions.
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.1
+    wcag_criterion: 1.3.1
+  - summary: Fields and labels are read in the same order that they appear on the page.
+    summary_additional: When you use a screen reader and navigate through file input labels and fields, you hear each text label and field announced in the same order in which they appear on the page.
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.1
+    wcag_criterion: 1.3.2
+  - summary: File input field announces purpose.
+    summary_additional: When you tab into the file input using a screen reader, you hear messaging that helps you identify where you need to input the file for upload.
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.1
+    wcag_criterion: 1.3.5
+  - summary: Screen reader announces all labels.
+    summary_additional: When you use a screen reader and navigate to a file input using arrow and tab keys, you hear labels and status messaging for selection state.
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.1
+    wcag_criterion: 2.4.6
+  - summary: Screen readers identify file input boxes the same way when more than one is on a page.
+    summary_additional: When you tab to a file input on a page that has other file inputs, you will hear "file input" every time you tab into a new one.
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.1
+    wcag_criterion: 3.2.4
+  - summary: Screen reader announces incorrect file type.
+    summary_additional: When you upload a file in the input box, the screen reader announces if you have uploaded the wrong file type.
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.1
+    wcag_criterion: 3.3.1
+  - summary: Screen reader announces errors. 
+    summary_additional: When you input content into a file input while using a screen reader, you will hear any error that occurs and a description of the error.
+    test_status: exception
+    status_details: Screen reader users who use the drag and drop feature may not hear the error state read aloud.
```

Comment:
@amycole501 @alex-hull 
This one feels redundant with 3.3.1 Wondering if we can merge this checklist item into 3.3.1, and then rewrite 4.1.3 to be more about giving all status updates, including when a file is uploaded successfully.

Reply:
@amyleadem The difference between this one and 4.1.3 is 3.3.1 is specific to text! When an error occurs, the user will see in text the error. Whether it announces to a screen reader is for 4.1.3.

Reply:
@amyleadem @amycole501 @alex-hull Double checking there is no edit needed here?

Reply:
@RachelCorsino This looks like the 4.1.3 criteria. We will be removing this test as a whole!